src/databricks/labs/ucx/workspace_access/redash.py

```diff
@@ -173,18 +169,9 @@ def hash_permissions(permissions: list[sql.AccessControl]):
                     f"doing another attempt..."
                 )
                 raise RetryableError(message=msg)
-        except DatabricksError as e:
-            if e.error_code in [
-                "BAD_REQUEST",
-                "UNAUTHORIZED",
-                "PERMISSION_DENIED",
-                "NOT_FOUND",
-            ]:
-                logger.warning(f"Could not update permissions for {object_type} {object_id} due to {e.error_code}")
-                return None
-            else:
-                msg = f"{e.error_code} can be retried for {object_type} {object_id}, doing another attempt..."
-                raise RetryableError(message=msg) from e
+        except NotFound:
+            logger.warning(f"Deleted on platform: {object_type} {object_id}")
```

Comment:
It's common practice to include the caught exception message into log along with context {object_type} {object_id}

Reply:
previous log messages didn't include that, out of scope of this PR